helm/ffc-demo-claim-service/templates/deployment.yaml

```diff
@@ -48,54 +48,54 @@ spec:
               protocol: TCP
           resources:
             requests:
-                memory: {{ .Values.container.requestMemory | quote }}
-                cpu: {{ .Values.container.requestCpu | quote }}
+                memory: {{ quote .Values.container.requestMemory }}
+                cpu: {{ quote .Values.container.requestCpu }}
             limits:
-                memory: {{ .Values.container.limitMemory | quote }}
-                cpu: {{ .Values.container.limitCpu | quote }}
+                memory: {{ quote .Values.container.limitMemory }}
+                cpu: {{ quote .Values.container.limitCpu }}
           env:
             - name: NODE_ENV
-              value: {{ .Values.environment }}
+              value: {{ quote .Values.environment }}
             - name: POSTGRES_USERNAME
-              value: {{ .Values.postgresUsername }}
+              value: {{ quote .Values.postgresUsername }}
             - name: POSTGRES_PASSWORD
-              value: {{ .Values.postgresPassword }}
+              value: {{ quote .Values.postgresPassword }}
             - name: POSTGRES_DB
-              value: {{ .Values.postgresDatabase }}
+              value: {{ quote .Values.postgresDatabase }}
             - name: POSTGRES_HOST
-              value: {{ .Values.postgresHost }}
+              value: {{ quote .Values.postgresHost }}
             - name: POSTGRES_PORT
-              value: {{ .Values.postgresPort | quote }}
+              value: {{ .Values.postgresPort }}
             - name: MESSAGE_QUEUE_HOST
-              value: {{ .Values.container.messageQueueHost }}
+              value: {{ quote .Values.container.messageQueueHost }}
             - name: MESSAGE_QUEUE_PORT
-              value: {{ .Values.container.messageQueuePort | quote}}
+              value: {{ .Values.container.messageQueuePort }}
```

Comment:
The Jenkins error suggests the port needs to be quoted.

Reply:
Yes it does.  All integers should not be quoted unless they are an environment variable which this is.